Makefile

```diff
@@ -98,7 +98,14 @@ inkscape:
 # register icon for Pd mimetype (adapted from https://stackoverflow.com/a/31836/1964109 )
 mime-register:
 	xdg-icon-resource install --context mimetypes --size 512 pd-512.png x-puredata
-	xdg-mime install x-puredata.xml 
+	xdg-mime install x-puredata.xml
+	cp pd-gui.desktop ~/.local/share/applications/pd-gui.desktop
+
+# 	following https://hoppenheit.info/blog/2016/where-to-put-application-icons-on-linux/
+install:
+	cp tweaked/pd.xpm /usr/share/pixmaps/puredata.xpm
+	cp pd-48.png /usr/share/icons/hicolor/48x48/apps/puredata.png
+	cp masters/icon.svg /usr/share/icons/hicolor/scalable/apps/puredata.svg
```

Comment:
same here: don't hardcode the paths.
- don't hardcode `/usr/share/`, as this is intended for distribution package managers; the default should be `/usr/local/share/` (if that works, i haven't tested) and should be overridable. typically with something like:
  ~~~Makefile
  prefix = /usr/local
  datarootdir = ${prefix}/share
  ~~~
  then use `${datarootdir}` as a replacemewnt for `/usr/share/`
- prefix all installation directories with `${DESTDIR}` (which should default to empty)

Reply:
Hmm. `/usr/local/share/pixmaps` doesn't exist on my system.